lib/backend/etcd/etcd.go

```diff
@@ -46,8 +46,12 @@ type EtcdClient struct {
 	etcdKeysAPI etcd.KeysAPI
 }
 
-func NewEtcdClient(config *capi.EtcdConfig) (*EtcdClient, error) {
-	// Determine the location from the authority or the endpoints.  The endpoints
+func getEtcdEndpoints(config *capi.EtcdConfig) ([]string, error) {
+	if (config.EtcdAuthority != "" || config.EtcdEndpoints != "") && config.EtcdDiscoverySrv != "" {
+		return nil, goerrors.New("multiple discovery or bootstrap options specified, use either \"etcdAuthority\"/\"etcdEndpoints\" or \"etcdDiscoverySrv\"")
```

Comment:
Could you tweak this message so that it doesn't include `etcdAuthority`, since that's deprecated and not recommended anymore.

Reply:
Done